app/services/audit/replication_support.rb

```diff
@@ -1,8 +1,8 @@
 # frozen_string_literal: true
 
 module Audit
-  # Catalog to cloud archive provider (what ZipEndpoint, ZippedMoabVersion etc represent) audit code
-  class CatalogToArchive
+  # Methods to support auditing replication.
+  class ReplicationSupport
```

Comment:
Why not Audit::ReplicatonService ... like every other service class?  "Support" is a new term being introduced, and I'm open to it if there's a reason and consistency.

Reply:
The reason I used `Support` was because in general (with the exception of `ActiveRecordUtils`) `Utils` in the codebase are standalone helpers for one-off work (e.g., via the rails console).